xarray/tests/test_backends.py

```diff
@@ -1139,6 +1257,8 @@ def test_dataarray_compute(self):
         self.assertTrue(computed._in_memory)
         self.assertDataArrayAllClose(actual, computed)
 
+class DaskTestAutocloseTrue(DaskTest):
+    autoclose=True
```

Comment:
Please run some sort of PEP8 check, e.g., `git diff upstream/master | flake8 --diff`. There should be extra spaces around the `=` sign here.